src/modules/paymentwall/controllers/front/ajax.php

```diff
@@ -0,0 +1,17 @@
+<?php
+class PaymentwallAjaxModuleFrontController extends ModuleFrontController
+{
+    public function postProcess()
+    {
+        if(!$_GET['orderId']) {
+            echo 0;
+            return;
+        }
+        $orderStatus = OrderHistory::getLastOrderState($_GET['orderId']);
+        if ($orderStatus) {
+            echo $orderStatus->id;
+        } else {
+            echo 0;
```

Comment:
```php
$orderId = Tools::getValue('orderId');
if(!$orderId || !$orderStatus = OrderHistory::getLastOrderState($orderId)) {
    echo 0;
} else {
    echo $orderStatus->id;
}
```
Please don't use global vars to get request params.
Because  Prestashop validator does not allow to use `$_GET` or `$_POST` in module.
Let's use `Tools::getValue` or `Tools::get`.